package.json

```diff
@@ -90,25 +90,19 @@
       "/.module-cache/",
       "/react/build/"
     ],
-    "persistModuleRegistryBetweenSpecs": true,
     "rootDir": "",
     "scriptPreprocessor": "scripts/jest/preprocessor.js",
-    "setupEnvScriptFile": "scripts/jest/environment.js",
+    "setupFiles": [ "scripts/jest/environment.js" ],
     "setupTestFrameworkScriptFile": "scripts/jest/test-framework-setup.js",
-    "testFileExtensions": [
-      "coffee",
-      "js",
-      "ts"
-    ],
+    "testRegex": "/__tests__/",
+    "moduleFileExtensions": ["js", "json", "node", "coffee", "ts"],
```

Comment:
Actually, this will get reprinted too (I think it's just`JSON.stringify(data, null, 2)`). Let's change this and the one above to have newlines, like it currently is.